.dockerignore

```diff
@@ -0,0 +1,2 @@
+artifacts/
```

Comment:
there are other directories that can be added here, but this is good starting to reduce the build time significantly